Description: Skip last part of test that is terminated by signal 9 on mips64el
Author: Doug Torrance <dtorrance@piedmont.edu>
Bug: https://github.com/Macaulay2/M2/issues/1663
Last-Update: 2020-12-06

--- a/M2/Macaulay2/tests/threads/schur-2.m2
+++ b/M2/Macaulay2/tests/threads/schur-2.m2
@@ -4,4 +4,4 @@
 S = schurRing(s,3);
 rep = s_{5};
 M = {1_S,s_{5},s_{10},s_{15},s_{20},s_{25},s_{30}};
-for i to 15 do ( stderr << i << endl; schurResolution(rep,M,SyzygyLimit => 3) );
+for i to 9 do ( stderr << i << endl; schurResolution(rep,M,SyzygyLimit => 3) );
